spot_tools/src/spot_executor/__init__.py

```diff
@@ -1,3 +1,3 @@
-#from spot_executor.executor_feedback_pyplot import FeedbackCollector
-#from spot_executor.fake_spot import FakeSpot
-#from spot_executor.spot_executor import SpotExecutor
+from spot_executor.executor_feedback_pyplot import FeedbackCollector
```

Comment:
@y-veys I had to comment these back in because they broke the ros executor. I think we need to merge this in to fix the current functionality, but we can move the action description stuff from ADT4 to this repo to fix your original issue.